airflow/executors/celery_executor.py

```diff
@@ -22,10 +22,14 @@
 
 class CeleryConfig(object):
     CELERY_ACCEPT_CONTENT = ['json', 'pickle']
-    CELERYD_PREFETCH_MULTIPLIER = 1
+    CELERYD_PREFETCH_MULTIPLIER = configuration.getint(
```

Comment:
Doing it this way will break existing installations that do not have this config variable defined. You probably want to add this to the defaults in configuration.py (set in case something is not present)